tests/functional/email_opt_in.js

```diff
@@ -76,90 +63,48 @@ define([
       // passed to basket. See a43061d3
       email = TestHelpers.createEmail('signup{id}+extra');
       return this.remote
-        .then(openPage(SIGNUP_PAGE_URL, '#fxa-signup-header'))
+        .then(openPage(SIGNUP_PAGE_URL, selectors.SIGNUP.HEADER))
         .then(fillOutSignUp(email, PASSWORD, { optInToMarketingEmail: true }))
 
-        .then(testElementExists('#fxa-confirm-header'))
+        .then(testElementExists(selectors.CONFIRM_SIGNUP.HEADER))
         .then(openVerificationLinkInSameTab(email, 0))
 
-        .then(testElementExists('#communication-preferences.basket-ready'))
+        .then(testElementExists(selectors.SETTINGS_COMMUNICATION.READY))
         .then(waitForBasket(email))
```

Comment:
This call fails when run locally because of https://github.com/mozilla/fxa-content-server/issues/5433